py3status/modules/watson.py

```diff
@@ -0,0 +1,87 @@
+"""
+Display the current status of the watson time-tracking tool.
+
+Configuration Parameters:
+    cache_time: Number of seconds before state is re-read (default: 10)
```

Comment:
```suggestion
    cache_timeout: Number of seconds before state is re-read
        (default 10)
```

cache_timeout is a standard to be respected on modules, please rename

when writing down default values, you have to get to a new line, indent, then not use ":" to stipulate the default